includes/templates/admin/edit-story.php

```diff
@@ -35,8 +35,9 @@
 
 // Preload common data.
 $preload_paths = [
-	sprintf( '/web-stories/v1/%s/%s?context=edit', $rest_base, $post->ID ),
+	sprintf( '/web-stories/v1/%s/%s?context=edit&_embed=author&_locale=user', $rest_base, $post->ID ),
```

Comment:
Pretty sure you do not need `&_locale=user`

Reply:
Embed requests like this will not work in preload, until this is https://github.com/google/web-stories-wp/pull/5149. 

Was this tested that this preloaded data worked?

Reply:
It actually doesn't work without `&_locale=user` when I tested.